tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -134,59 +134,79 @@ public function testFixProvider()
                 '<?php function xyz(
                     $a=10,      //comment1
                     $b=20,      //comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'must keep align comments (2)' => [
                 '<?php function xyz(
                     $a=10,  //comment1
                     $b=2000,//comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'multiline comments also must be ignored (I)' => [
-                '<?php function xyz(
+                <<<'EXPECTED'
+<?php function xyz(
```

Comment:
> different indent is intended

You stated that extra indent was intended, not mixed indent.

> That's why golden rule with utests states - add new tests, but don't modify existing ones.

I guess since my feature is optional now, I'm no longer forced to change these tests, so I'll revert them back to how they were written